src/components/scenes/LoginScene.tsx

```diff
@@ -152,9 +152,11 @@ class LoginSceneComponent extends React.PureComponent<Props, State> {
     showHelpModal()
   }
 
-  onComplete = () => {
-    this.props.navigation.navigate('gettingStarted', {})
-  }
+  onComplete = ENV.USE_WELCOME_SCREENS
+    ? () => {
+        this.props.navigation.navigate('gettingStarted', {})
+      }
+    : undefined
```

Comment:
It's purely preference, but I would have put the ternary at the JSX attribute instead of the definition of the method itself.

Reply:
maybeHandleComplete